WireAPI/Sources/WireAPI/APIs/UpdateEventsAPI/Event decoding/Team/TeamMemberLeaveEventDecoder.swift

```diff
@@ -41,7 +41,7 @@ struct TeamMemberLeaveEventDecoder {
         return TeamMemberLeaveEvent(
             teamID: teamID,
             userID: payload.userID,
-            time: time.date
+            time: time?.date ?? .now
```

Comment:
question: is the date even used ? or can we keep it optional?

Reply:
yes it is used when the user is removed from conversations, we append a system message (`func appendSystemMessage` from `ZMConversation`) that requires a non optional date. I've double checked on Android they decode this date from the event as well.